knowledge/geography/islands/prince_island/qna.yaml

```diff
@@ -0,0 +1,83 @@
+version: 2
+created_by: amaa1981, fatehks, webdba, youeyoue
+domain: geography
+task_description: |
+  Answer questions about Prince Leopold Island, located in Lancaster Sound at
+  the junction of Prince Regent Inlet and Barrow Strait in the Qikiqtaaluk Region
+  of Nunavut, Canada.
+seed_examples:
+- answer: The only resident land mammals are a small population of northern collared
+    lemmings, though visits by larger mammals - caribou, hares, foxes, and bears -
+    are recorded.
+  question: Does Prince Leopold Island have any resident mammals?
+- answer: Most of Prince Leopold Island rises steeply from the surrounding sea,
+    with cliffs rising as high as 265 m (869 ft) above sea level surrounding
+    most of the island.
+  question: How high are the cliffs surrounding Prince Leopold Island?
+- answer: The island and Somerset Island to its south were first sighted by a
+    European on August 4, 1819, during William Parry's 1819 expedition.
+  question: Prince Leopold Island was first sighted during which expedition?
+- answer: Large numbers of arctic seabirds, thick-billed murres, northern fulmars,
+    black-legged kittiwakes, and black guillemots breed on the cliff ledges, arriving
+    in the vicinity in May or early June and departing by mid-September.
+  question: What animals live on Prince Leopold Island?
+- answer: "74°N 90°W"
+  question: What are the coordinates of Prince Leopold Island?
+- answer: It is located in Lancaster Sound at the junction of Prince Regent
+    Inlet and Barrow Strait.
+  question: What body of water surrounds Prince Leopold Island?
+- answer: Prince Leopold Island is a significant breeding ground for large
+    populations of several arctic bird species.
+  question: For what species of animals is Prince Leopold Island a significant
+    summer habitat for?
+- answer: The majority of the island's area of 68 km2 (26 sq mi) is occupied by a
```

Comment:
This information is missing from the Markdown version

https://github.com/instructlab/taxonomy/pull/1154#issuecomment-2146437322